packages/core/src/services/commits/runDocumentAtCommit.ts

```diff
@@ -105,26 +116,35 @@ async function iterate({
     })
 
     for await (const value of streamToGenerator(result.fullStream)) {
-      controller.enqueue({
-        event: PROVIDER_EVENT,
+      enqueueEvent(controller, {
+        event: StreamEventTypes.Provider,
         data: value,
       })
     }
 
+    // TODO: The type on `ai` package return a different definition for `toolCalls`
+    // than `@latitude-data/compiler` package. We have to unify the naming.
+    // For now no toolCalls
```

Comment:
The naming in `ai` is not only ugly, but does not follow the openai schema.

Reply:
I don't understand this comment